cli/src/lib.rs

```diff
@@ -149,23 +151,29 @@ impl NodeBuilder {
         self
     }
 
+    /// Adds a service instance that will be available immediately after creating a genesis block.
+    ///
+    /// Make sure that artifact for this instance is provided to this builder.
+    pub fn with_default_instance(mut self, instance: impl Into<InstanceInitParams>) -> Self {
```

Comment:
I'd call this method `with_builtin_instance` (or `with_instance`, to be consistent with `TestKitBuilder`); see the reasoning above.